doc/scheduling.md

```diff
@@ -0,0 +1,180 @@
+# Scheduling in TempestaFW: User Guide
```

Comment:
Does Markdown provide support for contents? It'd be nice to have contents at begin of the file.